spyder/api/widgets.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+spyder.api.widgets
+==================
+
+This package exposes shareable widgets that can be used in
+third-party plugins.
+"""
+
+from spyder.widgets.tabs import Tabs
+from spyder.widgets.comboboxes import PythonModulesComboBox
+from spyder.widgets.variableexplorer.texteditor import TextEditor
```

Comment:
`from spyder.widgets.variableexplorer.texteditor import TextEditor` this is the only one that feels weird here because it imports from an actual "complex widget"